### DISCUSSION TITLE
Device Type > Add Components is broken

### DISCUSSION THREAD

**Discussion by ejm110** [netbox-community/netbox#6792] [Q&A] [answered]
As title says, cannot add any interface/console ports/etc using 'Add Components' on Device Type object as per picture below. But can add on each Device though.

Please help. Thank you in advance.

Netbox Version 2.11.8

![image](https://user-images.githubusercontent.com/40828218/126761694-3341e49f-f5f5-49db-bf74-cb389ec7aa9a.png)

**Comment by jeremystretch** [comment, marked as answer]
This was fixed in v2.11.9. See #6714 for more detail.
